dist/angular-swaggerific.js

```diff
@@ -2,7 +2,7 @@
     angular
         .module('angular-swaggerific', [])
         .factory('util', util)
-        .service('AngularSwaggerific', AngularSwaggerific);
+        .factory('AngularSwaggerific', AngularSwaggerific);
```

Comment:
You actually had it as a factory, I changed it to a service, and now changed it back because it caused a small annoyance. They're actually the exact same thing its just `factory` expects a return value, and `service` expects a constructor.